config/settings_data.json

```diff
@@ -1 +1,4 @@
-{}
+{
```

Comment:
Sorry Lucas, I accidentally saved in your editor store. Would you mind reverting once the changes are approved?